api/src/test/java/org/openmrs/aop/RequiredDataAdviceTest.java

```diff
@@ -34,9 +34,7 @@
 import org.junit.Assert;
 import org.junit.Before;
 import org.junit.Test;
-import org.mockito.Matchers;
-import org.mockito.Mock;
-import org.mockito.Spy;
+import org.mockito.*;
```

Comment:
Avoid these wild card imports, look here https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotusewildcardimports

Reply:
ok sir i will do the changes